docs/azure_arc_jumpstart/azure_arc_app_svc/aks/aks_azure_apimgmt_arm_template/_index.md

```diff
@@ -0,0 +1,185 @@
+  * _`myIpAddress`_ - Your local public IP address. This is used to allow remote RDP and SSH connections to the client Windows VM and AKS cluster.
+  * _`logAnalyticsWorkspaceName`_ - Unique name for the deployment log analytics workspace.
+  * _`kubernetesVersion`_ - AKS version
+  * _`dnsPrefix`_ - AKS unique DNS prefix
+  * _`deployAppService`_  Boolean that sets whether or not to deploy App Service plan, Azure Function application and Azure API Management. For this scenario, we leave it set to _**false**_.
+  * _`deployFunction`_ - Boolean that sets whether or not to deploy App Service plan, Azure Function application and Azure API Management. For this scenario, we leave it set to _**false**_.
+  * _`deployAPIMgmt`_ - Boolean that sets whether or not to deploy App Service plan, Azure Function application and Azure API Management.  For this scenario, we leave it set to _**true**_.
+  * _`templateBaseUrl`_ - GitHub URL to the deployment template - filled in by default to point to [Microsoft/Azure Arc](https://github.com/microsoft/azure_arc) repository, but you can point this to your forked repo as well.
+  * _`adminEmail`_ - an email address that will be used on the Azure API Management deployment to receive all system notifications.
+
+* To deploy the ARM template, navigate to the local cloned [deployment folder](https://github.com/microsoft/azure_arc/tree/main/azure_arc_app_services_jumpstart/aks/arm_template) and run the below command:
+
+    ```shell
+    az group create --name <Name of the Azure resource group> --location <Azure Region>
+    az deployment group create \
+    --resource-group <Name of the Azure resource group> \
+    --name <The name of this deployment> \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_app_services_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters <The *azuredeploy.parameters.json* parameters file location>
+    ```
+
+    > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the `azuredeploy.parameters.json` file**
+
+    For example:
+
+    ```shell
+    az group create --name Arc-AppSvc-Demo --location "East US"
+    az deployment group create \
+    --resource-group Arc-AppSvc-Demo \
+    --name arcappsvc \
+    --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_app_services_jumpstart/aks/arm_template/azuredeploy.json \
+    --parameters azuredeploy.parameters.json
+    ```
+
+    > **Note: The deployment time for this scenario can take ~60-90 minutes**
+
+* Once Azure resources has been provisioned, you will be able to see it in Azure portal. At this point, the resource group should have **7 various Azure resources** deployed.
+
+    ![ARM template deployment completed](./01.png)
+
+    ![New Azure resource group with all resources](./02.png)
```

Comment:
Resource Group name is no good. Use the same format we always and with no underscores